R/utilities.R

```diff
@@ -154,11 +156,13 @@ get_abundance_sc_wide = function(.data, features = NULL, all = FALSE, assay = .d
 #' @param features A character
 #' @param all A boolean
 #' @param exclude_zeros A boolean
+#' @param assay assay name to extract feature abundance
+#' @param slot slot in the assay, e.g. `data` and `scale.data`
 #'
 #' @return A Seurat object
 #'
 #' @export
-get_abundance_sc_long = function(.data, features = NULL, all = FALSE, exclude_zeros = FALSE){
+get_abundance_sc_long = function(.data, features = NULL, all = FALSE, exclude_zeros = FALSE, assay = .data@active.assay, slot = "data"){
```

Comment:
assay here is not needed as we will return all assays (inside the function is clear).